rerun_js/web-viewer/index.ts

```diff
@@ -115,6 +119,11 @@ export class WebViewer {
     this.#canvas.id = this.#id;
     parent.append(this.#canvas);
 
+    // This yield appears to be necessary to ensure that the canvas is attached to the DOM
+    // and visible. Without it we get occasionally get a panic about a failure to find a canvas
+    // element with the given ID.
+    await delay(0);
```

Comment:
@jprochazk I hit this issue again after bringing up the local-server variant. Still seems like some kind of DOM-state synchronization issue.

Reply:
Would be nice if egui supported receiving a `HTMLCanvasElement` directly instead of an `id` which it uses to find it in the document.

Reply:
Opened an issue in egui: https://github.com/emilk/egui/issues/4746